setup.py

```diff
@@ -128,10 +129,12 @@
     "GitPython<3.1.19",
     "scipy",
     "onnx",
+    "torchprofile>=0.0.4",
     "optimum_quanto>=0.2.6",
     "gguf>=0.10.0",
     "torchao>=0.7.0",
     "bitsandbytes>=0.43.3",
+    "nvidia_modelopt>=0.27.0",
```

Comment:
```suggestion
    "nvidia_modelopt[torch]>=0.27.0",
```
This will add pulp, torchprofile and other future dependencies automatically